demos/device_defender_for_aws/report_builder.h

```diff
@@ -62,8 +72,8 @@ typedef struct ReportMetrics
  * @param[in] pMetrics Metrics to write in the generated report.
  * @param[in] majorReportVersion Major version of the report.
  * @param[in] minorReportVersion Minor version of the report.
- * @param[in] reportId Value to be used as the reportId in the generated report.
- * @param[out] pOutReprotLength The length of the generated report.
+ * @param[in] reportId Value to be used as the ulReportId in the generated report.
```

Comment:
What does `ulReportId` refer to? If it is a variable internal to the report builder implementation, then it shouldn't be mentioned in the API.

Reply:
Oops, missed changing that while changing the style. Though that is a json field so probs needs to be fixed in FreeRTOS/FreeRTOS